Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -294,32 +294,19 @@ private func backerCountOrRemainingString(project: Project, reward: Reward) -> R
 }
 
 private func shippingSummaryString(project: Project, reward: Reward) -> RewardCardPillData? {
-  if project.state == .live, reward.shipping.enabled, let type = reward.shipping.type {
-    switch type {
-    case .anywhere:
-      return RewardCardPillData(
-        backgroundColor: UIColor.ksr_create_700.withAlphaComponent(0.06),
-        text: Strings.Ships_worldwide(),
-        textColor: UIColor.ksr_create_700
-      )
-    case .multipleLocations:
-      return RewardCardPillData(
-        backgroundColor: UIColor.ksr_create_700.withAlphaComponent(0.06),
-        text: Strings.Limited_shipping(),
-        textColor: UIColor.ksr_create_700
-      )
-    case .noShipping: return nil
-    case .singleLocation:
-      if let name = reward.shipping.location?.localizedName {
-        return RewardCardPillData(
-          backgroundColor: UIColor.ksr_create_700.withAlphaComponent(0.06),
-          text: Strings.location_name_only(location_name: name),
-          textColor: UIColor.ksr_create_700
-        )
-      }
-
-      return nil
-    }
+  if project.state == .live,
```

Comment:
does this mean this info is hidden when the project is not live?

Reply:
Hey Bessie, yes, the project needs to be in `live` state to show the shipping labels. 
For all the base reward cards, (add-ons are only shown if project is live) we show 2 strings when the project isn't live (backer count and has add-ons) and two more if the project is live (shipping and time left).
Confirmed its the same on Android for non-line projects.
![Screen Shot 2022-05-17 at 1 44 04 PM](https://user-images.githubusercontent.com/4282741/168876925-5a593d34-80c1-431c-b25e-6e06a75cefff.png)

Reply:
thanks for clarifying!